src/styles/main.scss

```diff
@@ -1 +1,90 @@
 @use "@nationalarchives/frontend/nationalarchives/all";
+
+.development-notice {
+  padding: 8px;
+
+  background-color: var(--background, #f4f4f4);
+
+  border: 0.2em var(--form-error-border, #937ef1) solid;
+  border-radius: 6px;
+}
+
+.choose-section {
+  background-color: #dde5d5;
+}
```

Comment:
I would remove this and add a class of `tna-block-accent-light` to the element instead (https://design-system.nationalarchives.gov.uk/styles/colours/#block-colours)